services-directions-models/src/test/java/com/mapbox/api/directions/v5/models/IncidentTest.java

```diff
@@ -0,0 +1,88 @@
+package com.mapbox.api.directions.v5.models;
+
+import com.google.common.collect.Lists;
+import com.mapbox.core.TestUtils;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class IncidentTest extends TestUtils {
+
+  @Test
+  public void sanity(){
+    assertNotNull(getDefault());
+  }
+
+  @Test
+  public void testSerializable() throws Exception {
```

Comment:
Same as above

> Minor thing: We should remove `test` as it's implicit from `@Test`. Could we add a better naming reflecting what's being tested here?

Reply:
https://github.com/mapbox/mapbox-java/pull/1201#discussion_r502018321

Reply:
`testSerializable` and `serializableTest` 🤯

Reply:
renamed to `testSerializableObject`